README.md

```diff
@@ -1 +1,56 @@
-# Conference-App
\ No newline at end of file
+# capstone project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Conference page
+  - This is my Conference page. With the proper flow of git and Linter I make this page.
+  Firstly I make mobile website using basic HTML ans CSS. After that I worked on bigger screen like desktop and with 
+  Adjusticable width I work on desktop. After completing the desktop I work on js for making the mobile menu  appear.
+  After I make the website dymanic using js.
+
+## Built With
+
```

Comment:
- [ ] In your video, you did not present any project. You simply talked about what you did. Please record another video and showcase your project by talking through what you did.